cli/args.js

```diff
@@ -68,7 +69,8 @@ export function help() {
     --no-overwrite will not run generation on Content with an existing Response.
     --summary will show a pricing expectation before running and prompt for OK.
     -y, —-yes will skip any prompts.
-    -v, --verbose, --log-level v and verbose will set log level to info; --log-level can be a string or number and use jefri/jiffies logging levels.
+    -v, --verbose, --log-level v and verbose will set log level to info; --log-level can be a string or number and use jefri/jiffies logging levels. Ailly uses warn for reporting details on errors, info for general runtime progress, and debug for details of requests and responses. Logs are structures in JSONL.
```

Comment:
`JSONL` typo?

Reply:
Nope, it's [JSONL](https://jsonlines.org)